src/detect-flow-pkts.c

```diff
@@ -23,238 +23,412 @@
 #include "detect-engine-uint.h"
 #include "detect-parse.h"
 
-static int DetectFlowPktsToClientMatch(
+enum FlowDirection {
+    DETECT_FLOW_TOSERVER = 1,
+    DETECT_FLOW_TOCLIENT,
+    DETECT_FLOW_TOEITHER,
+};
+
+typedef struct DetectFlow_ {
```

Comment:
Why do we share the same structure for packets and bytes ?

(also would be nice to save one alloc)